packages/node/src/types.ts

```diff
@@ -170,6 +170,16 @@ export interface WithTrustRoot {
   trustRoot?: boolean
 }
 
+export interface ScuttleGlobalThis {
```

Comment:
```suggestion
export interface ScuttleGlobalThisOptions {
```

I realize that this demands changes to the reference so is not easily just done thru the UI

Reply:
like this? 6a560735a